custom_components/frigate/media_source.py

```diff
@@ -136,7 +136,7 @@ def get_identifier_type(cls) -> str:
 
     def get_frigate_server_path(self) -> str:
         """Get the equivalent Frigate server path."""
-        return "/".join((self.get_identifier_type(), self.name))
+        return f"vod/event/{self.name}/index.m3u8"
```

Comment:
(Apologies I don't know about this new end-point)

Will pre-existing clips continue to be accessible via this new path? If not we should probably use a different identifier type.

Reply:
yes, this endpoint first checks if a video exists under `/media/frigate/clips/{camera}-{event_id}.mp4` and will return that. Otherwise it will generate the clip from the recording files.

Everything is currently fully backwards-compatible.